addons/mail/static/tests/qunit_suite_tests/components/chatter_tests.js

```diff
@@ -260,6 +260,35 @@ QUnit.test("chatter: drop attachments", async () => {
     await contains(".o_AttachmentCard", { count: 3 });
 });
 
+QUnit.test("error on uploading file in chatter shows error", async function (assert) {
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({});
+    const { openView } = await start({
+        async mockRPC(route) {
+            if (route === '/mail/attachment/upload') {
+                throw new Error("cannot upload this file");
+            }
+        }
+    });
+    openView({
```

Comment:
Prefer to always `await`.

Can't use `openFormView`? I don't remember if it exists in this version.

Reply:
change done